tests/kernel/common/src/main.c

```diff
@@ -20,6 +20,14 @@ extern void test_timeout_order(void);
 extern void test_clock_cycle(void);
 extern void test_clock_uptime(void);
 extern void test_multilib(void);
+
+/**
+ * @defgroup kernel_common_tests common Tests
+ * @ingroup all_tests
+ * @{
```

Comment:
when you do this, everything is added to doxygen, also non-tests, either move the helper functions out of the brackets or use `@ingroup kernel_common_tests` individually on each documented test. Here for example delayed_resubmit_work_handler  needs to be moved out or excluded.

Reply:
@nashif : I think there is no such helper functions to be moved out. All are test functions.